Quaver.Shared/Screens/Edit/UI/Menu/EditorFileMenuBar.cs

```diff
@@ -347,6 +348,14 @@ private void CreateViewSection()
 
                 foreach (var map in Screen.Map.Mapset.Maps)
                 {
+                    try
+                    {
+                        File.OpenRead(Directory.GetCurrentDirectory() + "/Songs/" + map.Directory + "/" + map.Path);
+                    } catch (FileNotFoundException)
+                    {
+                        break;
+                    }
```

Comment:
This isn't something you're going to want to do every single frame. If the goal is to fix the crash when attempting to load the file, then that should be handled when the user clicks it (and display a notification).

Reply:
Found another way and fixed with https://github.com/Quaver/Quaver.API/pull/117